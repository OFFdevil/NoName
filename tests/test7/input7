def FUNCa A
{
    // body 1
}

def FUNCa A
{
    // body 2
}

main
  (A)&&(A);