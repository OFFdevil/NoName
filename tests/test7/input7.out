Function: name= FUNCa has already created with such parametrs!
